pyproject.toml

```diff
@@ -1,2 +1,58 @@
 [tool.black]
 target-version = ['py310']
+
+[tool.poetry]
+name = "screenpy"
+version = "4.0.2"
+description = "Screenplay pattern base for Python automated UI test suites."
+authors = ["Perry Goy <perry.goy@gmail.com>"]
+license = "MIT"
+#packages = []
+repository = "https://github.com/ScreenPyHQ/screenpy"
+documentation = "https://screenpy-docs.readthedocs.io"
+readme = "README.md"
+classifiers = [
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Quality Assurance",
+    "Topic :: Software Development :: Testing",
+    "Topic :: Software Development :: Testing :: BDD",
+    "Natural Language :: English",
+    "License :: OSI Approved :: MIT License",
+]
+
+[tool.poetry.dependencies]
+python = "^3.8"
```

Comment:
It occurs to me that we're a little loose with our supporting of python versions.  It might be a good idea to finally add the github actions to test each supported version. (whatever we decide here)

Reply:
At least on my machine, i run the tests in all our supported environments using `tox` (which is also mentioned in the "Contributing" section of the README. :P). It runs all our unit tests through each version of Python listed in its `.ini`.

Reply:
After this PR, I have another branch to merge that runs the tests in multiple versions of python as well.  I seem to recall a discussion about dropping 3.7 support for some reason (though I cannot remember what is was)

That said, tests and linters appears to pass in 3.7.  That either means 3.7 works just fine, or our tests are incomplete to show 3.7 shouldn't be working.😬

Reply:
I also noticed that the supported versions reported by pypi is just 3. 
[![Supported Versions](https://img.shields.io/pypi/pyversions/screenpy.svg)](https://pypi.org/project/screenpy)

I assume it gets that from `"Programming Language :: Python :: 3 :: Only",`.  It feels like a good idea to have a consistent reporting of supported versions between pypi, tox, github actions, etc.